modules/nf-core/stadeniolib/cramfilter/main.nf

```diff
@@ -0,0 +1,34 @@
+
+process STADENIOLIB_CRAMFILTER {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::staden_io_lib=1.14.14"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/staden_io_lib:1.14.14--h0d9da7e_3':
+        'biocontainers/staden_io_lib:1.14.14--h0d9da7e_3' }"
+
+    input:
+    tuple val(meta), path(incram), path(cramIdx)
+    val from
+    val to
```

Comment:
Wouldn't it be better to merge these inputs into `interval` for example? This would make it clearer in my opinion from a user standpoint IMO